app/models/atmosphere/deployment.rb

```diff
@@ -10,10 +10,24 @@
 # Deployments provide a m:n link between appliances and virtual machines.
 module Atmosphere
   class Deployment < ActiveRecord::Base
+    extend Enumerize
     belongs_to :appliance,
       class_name: 'Atmosphere::Appliance'
 
     belongs_to :virtual_machine,
       class_name: 'Atmosphere::VirtualMachine'
+
+    validates :billing_state, presence: true
+
+    enumerize :billing_state, in: ["initial", "prepaid", "expired", "error"], predicates: true
```

Comment:
Line is too long. [94/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.